maintainers/maintainer-list.nix

```diff
@@ -842,6 +842,12 @@
     githubId = 354741;
     name = "Austin Butler";
   };
+  avanov = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add avanov``.